tests/layer_tests/tensorflow2_keras_tests/test_tf2_keras_add.py

```diff
@@ -1,11 +1,9 @@
-# Copyright (C) 2022 Intel Corporation
+# Copyright (C) 2022-2024 Intel Corporation
```

Comment:
I saw some files have 2018-2024. Which is correct?

Reply:
both versions are correct. It depends in what year file is created.